modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/WebDataFile_conf.pm

```diff
@@ -0,0 +1,161 @@
+                           run_all        => $self->o('run_all'),   
+                         },
+          -flow_into   =>  {  '2->A' => ['StepBootstrap'],
+                              '3->A'  => ['SpeciesFactory'],   
+                               # WHEN(
+                               #   '#genomeinfo_yml#' =>  ['StepBootstrap'],
+                               #    ELSE                  ['SpeciesFactory'],
+                               # ),
+                              'A->1' => ['email_notification'],   
+                           }
+
+        }, 
+
+        {
+            -logic_name => 'SpeciesFactory',
+            -module     =>
+                'Bio::EnsEMBL::Production::Pipeline::Common::SpeciesFactory',
+            #-input_ids  => [ {} ], # required for automatic seeding
+            -parameters => { species => $self->o('species'),
+                antispecies          => $self->o('antispecies'),
+                division             => $self->o('division'),
+                run_all              => $self->o('run_all') 
+             },
+            -flow_into  => { '2' => [ 'StepBootstrap' ],
+                             #'A->1' => ['email_notification']
+                           },
+             
+        },
+        {    
+            -logic_name => 'StepBootstrap',
+            -module     => 'Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFile',
+            -parameters => { current_step       => 'bootstrap',
+                             step => $self->o('step',)
+                           }, 
+            -flow_into        => {
+                      '1' =>  ['StepGeneAndTranscript'],
+                      '2' =>  ['StepContigs'],
+                      '3' =>  ['StepGC'],
+                      '4' =>  ['StepVariation'],
+            },
+
+        },
+        {   -logic_name => 'StepGeneAndTranscript',
+              -module     => 'Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFile',
+        },
+        {   -logic_name => 'StepContigs',
+              -module     => 'Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFile',
+        },
+        {   -logic_name => 'StepGC',
+            -module     => 'Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFile',
+	},
+        {   -logic_name => 'StepVariation',
+            -module     => 'Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFile',
+        },
+        {
+           -logic_name => 'email_notification',
+           -module     => 'Bio::EnsEMBL::Hive::RunnableDB::NotifyByEmail',
+           -parameters => {
+                           'email'   => $self->o('email'),
+                           'subject' => 'Webdatafile pipeline  completed',
+                           'text'    => 'Output location : '. $self->o('output_path'), 
+                          },
+        },
+
+ 
+
+    ];
+} ## end sub pipeline_analyses
+
+sub beekeeper_extra_cmdline_options {
```

Comment:
Don't think this is needed anymore. see comment from @james-monkeyshines  here: https://github.com/Ensembl/ensembl-production/pull/452/files#r456581282

Reply:
base package is added as per suggestion